src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -0,0 +1,828 @@
+Apple accommodates for this user preference with a feature called
+[Smart Invert](https://www.apple.com//accessibility/iphone/vision/),
+which reverses the colors on the display, except for images, media,
+and some apps that use dark color styles.
+
+A special form of low vision is Computer Vision Syndrome, also known as Digital Eye Strain, which is
+[defined](https://onlinelibrary.wiley.com/doi/full/10.1111/j.1475-1313.2011.00834.x)
+as *“the combination of eye and vision problems associated with the use of computers
+(including desktop, laptop, and tablets) and other electronic displays (e.g.
+smartphones and electronic reading devices).”*
+It has been [proposed](https://bmjopen.bmj.com/content/5/1/e006748)
+that the use of electronic devices by adolescents, particularly at night time,
+leads to an increased risk of shorter sleep duration,
+longer sleep-onset latency, and increased sleep deficiency.
+Additionally, exposure to blue light has been widely
+[reported](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4254760/)
+to be involved in the regulation of
+[circadian rhythm](https://en.wikipedia.org/wiki/Circadian_rhythm)
+and the sleep cycle,
+and irregular light environments may lead to sleep deprivation,
+possibly affecting mood and task performance, according to
+[research by Rosenfield](https://www.college-optometrists.org/oip-resource/computer-vision-syndrome--a-k-a--digital-eye-strain.html).
+To limit these negative effects, reducing blue light by adjusting the display color temperature
+through features like iOS’ [Night Shift](https://support.apple.com/en-us/HT207570) or Android’s
+[Night Light](https://support.google.com/pixelphone/answer/7169926?) can help,
+as well as avoiding bright lights or irregular lights in general through dark themes or dark modes.
+
+#### Dark mode power savings on AMOLED screens
+
+Finally, dark mode is known to save a *lot* of energy on
+<abbr title="Active-Matrix Organic Light-Emitting Diode">AMOLED</abbr> screens.
+Android case studies that focused on popular Google apps
+like YouTube have shown that the power savings can be up to 60%.
+The video below has more details on these case studies and the power savings.
+
+<figure class="w-figure w-figure--fullbleed">
+  {% YouTube 'N_6sPd0Jd3g?start=305' %}
+</figure>
+
+## Activating dark mode in the operating system
+
+Now that I have covered the background of why dark mode is such a big deal for many users,
+let’s review how you can support it.
+
+Operating systems that support a dark mode or dark theme
+typically have an option to activate it somewhere in the settings.
+On macOS&nbsp;X, it’s in the system preference’s *General* section and called *Appearance*,
+and on Windows&nbsp;10, it’s in the *Colors* section and called *Choose your color*.
+For Android&nbsp;Q, you can find it under *Display* as a *Dark Theme* toggle switch,
+and on iOS&nbsp;13, you can change the *Appearance* in the *Display &amp; Brightness*
+section of the settings.
+
+<figure>
```

Comment:
We've discouraged folks from doing custom CSS on the site because it will likely break as we rev the design in the future.

In this case the screenshots don't seem critical so it might be worth replacing this with a single screenshot as an example.

Reply:
I have removed the screenshots and just left one image in https://github.com/GoogleChrome/web.dev/pull/1021/commits/9cd638b5c40b640fa70b2dec87964148d895e779.